docs/team/app-guests.md

```diff
@@ -0,0 +1,43 @@
+title: Sematext App Guests
+description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.
```

Comment:
description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user for example, dashboards are at the account level can can thus be shared only through Account Sharing.

=>

description: App Sharing is restricted only to a particular App. Nothing besides the shared App is accessible to the invited user. For example, Dashboards are at the account level can can thus be shared only through Account Sharing.